easybuild/easyconfigs/c/CLHEP/CLHEP-2.1.3.1-intel-2015a.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'CMakeMake'
+
+name = 'CLHEP'
+version = '2.1.3.1'
+
+homepage = 'http://proj-clhep.web.cern.ch/proj-clhep/'
+description = """The CLHEP project is intended to be a set of HEP-specific foundation and
+ utility classes such as random generators, physics vectors, geometry and linear algebra.
+ CLHEP is structured in a set of packages independent of any external package."""
+
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'pic': True}
+
+sources = [SOURCELOWER_TGZ]
+source_urls = ['http://proj-clhep.web.cern.ch/proj-clhep/DISTRIBUTION/tarFiles/']
+
+dependencies = [('CMake', '3.1.3')]
+
+# build dir should not be in the source tree!
+ext_bldd = '../../builddir'
+preconfigopts = 'mkdir %s && cd %s &&' % (ext_bldd,ext_bldd)
+prebuildopts = 'cd %s &&' % ext_bldd
+preinstallopts = 'cd %s &&' % ext_bldd
+
+configopts = '-DCMAKE_BUILD_TYPE=Release' 
+
+# build dir should not be in the source tree!
+srcdir = '../%s/%s/' % (version,name)
```

Comment:
you can probably drop this too if you change it to using `separate_build_dir = True`